features/counters.yml.dist

```diff
@@ -0,0 +1,96 @@
+# Generated from: counters.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2015-07-29
+  baseline_high_date: 2018-01-29
+  support:
+    chrome: "1"
+    chrome_android: "18"
+    edge: "12"
+    firefox: "1"
+    firefox_android: "4"
+    safari: "3"
+    safari_ios: "1"
+compat_features:
+  # ⬇️ Same status as overall feature ⬇️
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "4"
+  #   safari: "3"
+  #   safari_ios: "1"
+  - css.types.counter
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1.5"
+  #   firefox_android: "4"
+  #   safari: "3"
+  #   safari_ios: "1"
+  - css.types.counters
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "25"
+  #   safari: "3"
+  #   safari_ios: "1"
+  - css.properties.counter-increment
+  - css.properties.counter-reset
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "25"
+  #   safari: "3"
+  #   safari_ios: "2"
+  - css.properties.counter-increment.none
+  - css.properties.counter-reset.none
+
+  # baseline: high
+  # baseline_low_date: 2020-05-19
+  # baseline_high_date: 2022-11-19
+  # support:
+  #   chrome: ≤83
+  #   chrome_android: "83"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "25"
+  #   safari: ≤13.1
+  #   safari_ios: ≤13.4
+  - css.properties.counter-increment.list-item
+  - css.properties.counter-reset.list-item
+
+  # baseline: false
+  # support:
+  #   firefox: "82"
+  #   firefox_android: "82"
+  - css.properties.counter-reset.reset_does_not_affect_siblings
+
+  # baseline: false
+  # support:
+  #   firefox: "96"
+  #   firefox_android: "96"
+  - css.properties.counter-reset.reversed
```

Comment:
`reset_does_not_affect_siblings` seems mostly like bringing the implementation into compliance with the spec in an edge case, and I don't think it deserves a feature.

`reversed` is an interesting thread to pull at. Thinking through what that feature would be, I think the feature would be something like "Reversed lists", and we would likely want to pull `html.elements.ol.reversed` out of the `list-items` feature, which has a different support story.

There are blogs and questions about how to reverse lists, but I'm not sure if it raises to the level of feature.

Reply:
I'd love to hear a third opinion on this. I could go both ways, but slight preference for splitting reversed lists into its own feature.

Reply:
@ddbeck  or @mirisuzanne for a third opinion - do you think "Reversed lists" should be a separate feature? It would include `css.properties.counter-reset.reversed` (only supported in Firefox) and `html.elements.ol.reversed` (Baseline High), and probably be computed from `html.elements.ol.reversed`.

Reply:
In an ideal world, we'd have a "reversed list" feature alongside the property and element features. We can't do that just yet (see https://github.com/web-platform-dx/web-features/issues/1173), so I'm suggesting the following:

- Split `counter-reset: reversed` CSS declaration into it's own feature (you can take inspiration from one the `contain-` features, for descriptions)
- Put a comment in that feature mentioning that, after https://github.com/web-platform-dx/web-features/issues/1173 is resolved, that a "reversed list" feature would be a good addition, crossing the CSS and HTML boundary)

Reply:
I think splitting it out is also the best option for making the support data visible - since the support story for `counter-reset.reversed` doesn't match either of the other groupings. To me, at least, it seems like placing an unsupported key in with supported keys only acts to 'hide' the browser support that this project is meant to surface.